fuzzy/fuzzy.go

```diff
@@ -150,6 +156,9 @@ type Rank struct {
 
 	// Distance is the Levenshtein distance between Source and Target.
 	Distance int
+
+	// Location of Target in original list
+	Index int
```

Comment:
I think the naming could be a potential cause for confusion. I expect many to assume `Index` here means the index in the filtered list, not the original one. `OriginalIndex` maybe...? Naming is hard :)